models/sap.yml

```
@@ -1,11 +1,975 @@
+        description: Cross-distribution-chain material status
+      - name: mstde
+        description: Date from which the cross-plan material status is valid
+      - name: mstdv
+        description: Date from which the X-distr.-chain material status is valid
+      - name: taklv
+        description: Tax classification of the material
+      - name: rbnrm
+        description: Catalog profile
+      - name: mhdrz
+        description: Minimum remaining shelf life
+      - name: mhdhb
+        description: Total shelf life
+      - name: mhdlp
+        description: Storage percentage
+      - name: inhme
+        description: Content unit
+      - name: inhal
+        description: Net contents
+      - name: vpreh
+        description: Comparison price unit
+      - name: etiag
+        description: IS-R labeling (material grouping)
+      - name: inhbr
+        description: Gross contents
+      - name: cmeth
+        description: Quantity conversion method
+      - name: cuobf
+        description: Internal object number
+      - name: kzumw
+        description: Environmentally relevant
+      - name: kosch
+        description: Product allocation determination procedure
+      - name: sprof
+        description: Pricing profile for variants
+      - name: nrfhg
+        description: Material qualifies for discount in kind
+      - name: mfrpn
+        description: Manufacturer part number
+      - name: mfrnr
+        description: Manufacturer number
+      - name: bmatn
+        description: Number of firm's own (internal) inventory-managed material
+      - name: mprof
+        description: Mfr part profile
+      - name: kzwsm
+        description: Units of measure usage
+      - name: saity
+        description: Rollout in a season
+      - name: profl
+        description: Dangerous goods indicator profile
+      - name: ihivi
+        description: Indicator (highly viscous)
+      - name: iloos
+        description: Indicator (in bulk/liquid)
+      - name: serlv
+        description: Level of explicitness for serial numbers
+      - name: kzgvh
+        description: Packaging material is closed packaging
+      - name: xgchp
+        description: Indicator (Approved batch record required)
+      - name: kzeff
+        description: Assign effectivity parameter values / override change numbers
+      - name: compl
+        description: Material completion level
+      - name: iprkz
+        description: Period indicator for shelf life expiration date
+      - name: rdmhd
+        description: Rounding rule for calculation of SLED
+      - name: przus
+        description: Indicator (Product composition printed on packaging)
+      - name: mtpos_mara
+        description: General item category group
+      - name: bflme
+        description: Generic material with logistical variants
+      - name: nsnid
+        description: NATO item identification number
+
+  - name: sap__0vendor_attr
+    description: Vendor attributs
```

Request for more details in description of the model.

```suggestion
    description: Vendor attributes
```

---

Updated